spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java

```diff
@@ -183,7 +183,9 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
 
     if (sparkInterpreter != null && sparkInterpreter.isSparkContextInitialized()) {
       return new InterpreterResult(Code.ERROR,
-          "Must be used before SparkInterpreter (%spark) initialized");
+          "Must be used before SparkInterpreter (%spark) initialized\n" +
+          "Hint: put this cell before any spark/scala code or/and " +
+          "restart Zeppelin" );
```

Comment:
@r-kamath - I'm wondering is "paragraph" some widely used term also in other notebooks (Jupyter), or is this word Zeppelin specific? I mean it is easy to change, just let's agree this is the right word ...